spec/features/brexit_checker/brexit_checker_spec_helper.rb

```diff
@@ -0,0 +1,8 @@
+module BrexitCheckerSpecHelper
```

Comment:
Thanks @koetsier +1. Helpers generally go in `spec/support`, though.

Reply:
👍 saves a ```require``` as well